app/navigation/nonsipp/TotalValueQuotedSharesNavigator.scala

```diff
@@ -16,19 +16,31 @@
 
 package navigation.nonsipp
 
-import models.UserAnswers
+import models.{CheckMode, NormalMode, UserAnswers}
 import navigation.JourneyNavigator
 import pages.Page
-import pages.nonsipp.totalvaluequotedshares.TotalValueQuotedSharesPage
+import pages.nonsipp.totalvaluequotedshares._
 import play.api.mvc.Call
 
 object TotalValueQuotedSharesNavigator extends JourneyNavigator {
 
-  override def normalRoutes: UserAnswers => PartialFunction[Page, Call] = userAnswers => {
+  override def normalRoutes: UserAnswers => PartialFunction[Page, Call] = _ => {
 
     case TotalValueQuotedSharesPage(srn) =>
+      controllers.nonsipp.totalvaluequotedshares.routes.TotalValueQuotedSharesCYAController.onPageLoad(srn, NormalMode)
+
+    case TotalValueQuotedSharesCYAPage(srn) =>
```

Comment:
@kuldeepwadhwa9 If you decide to remove checkRoutes below, I think this line would handle navigation back to the task list and resolve the issue highlighted on the ticket.